frontend/public/actions/features.ts

```diff
@@ -156,6 +156,18 @@ const detectUser = dispatch => coFetchJSON('api/kubernetes/apis/user.openshift.i
     },
   );
 
+const detectConsoleLinks = dispatch => coFetchJSON('api/kubernetes/apis/console.openshift.io/v1/consolelinks')
+  .then(
```

Comment:
@spadgett @rhamilto curious if this has been tested as a non-clusteradmin/kubeadmin user.  I don't think normal users will be able to see these.

Reply:
We will need to add a role binding for authenticated users to list/get/watch console links

Reply:
RBAC added here: https://github.com/openshift/console-operator/pull/229/files#diff-917b39b291003c5a9d8a57afceaa65aaR68
With an objection, but no surprise: https://github.com/openshift/console-operator/pull/229/files#r288233402